pkg/provision/providers/qemu/launch.go

```diff
@@ -257,6 +258,18 @@ func launchVM(config *LaunchConfig) error {
 		"-smbios", fmt.Sprintf("type=1,uuid=%s", config.NodeUUID),
 	}
 
+	if config.QEMUGuestSocketEnabled {
+		args = append(
```

Comment:
should we just always create the guest agent socket? is there any harm in enabling it?

Reply:
no, it's fine, I'll update then

Reply:
I mean we could simplify by having it enabled always, one less flag to support :-D